app/shared/connectors/httpparsers/HttpParser.scala

```diff
@@ -53,22 +53,27 @@ trait HttpParser extends Logging {
 
   private val multipleErrorReads: Reads[Seq[DownstreamErrorCode]] = (__ \ "failures").read[Seq[DownstreamErrorCode]]
 
+  private val multipleHipErrorReads: Reads[Seq[HipDownstreamErrorCode]] = (__ \ "failures").read[Seq[HipDownstreamErrorCode]]
+
   private val bvrErrorReads: Reads[Seq[DownstreamErrorCode]] = {
     implicit val errorIdReads: Reads[DownstreamErrorCode] = (__ \ "id").read[String].map(DownstreamErrorCode(_))
     (__ \ "bvrfailureResponseElement" \ "validationRuleFailures").read[Seq[DownstreamErrorCode]]
   }
 
   def parseErrors(response: HttpResponse): DownstreamError = {
     val singleError         = response.validateJson[DownstreamErrorCode].map(err => DownstreamErrors(List(err)))
+    val singleHipError      = response.validateJson[HipDownstreamErrorCode].map(err => DownstreamErrors(List(err.toDownstreamErrorCode)))
```

Comment:
I don't think this is needed, errors will always be contained in the failures array